ios/RCTFBSDK/login/RCTFBSDKLoginButtonManager.m

```diff
@@ -45,6 +45,14 @@ - (UIView *)view
 
 RCT_EXPORT_VIEW_PROPERTY(defaultAudience, FBSDKDefaultAudience)
 
+ RCT_CUSTOM_VIEW_PROPERTY(nonceIOS, NSString, FBSDKLoginButton){
+   [view setNonce:json ? json : nil];
+ }
+
+ RCT_CUSTOM_VIEW_PROPERTY(loginTrackingIOS, NSString, FBSDKLoginButton){
+   [view setLoginTracking:(json && [json  isEqual: @"limited"]) ? FBSDKLoginTrackingLimited : FBSDKLoginTrackingEnabled];
```

Comment:
Here, since we know `json` is a string, it is faster to use `-[NSString isEqualToString:]` method, so `[json isEqualToString:@"limited"]`